josue/js/app.js

```diff
@@ -1,37 +1,49 @@
 import FormValidator from './Formvalidator.js';
 
-const form = document.querySelector('.form');
+const form = document.querySelector('#form');
 const signInBtn = document.querySelector("#signBtn");
 const registerBtn = document.querySelector("#registerBtn");
 const submitBtn = document.querySelector("#submit");
-const registerFields = document.querySelector("#register-fields");
+const registerFieldsWrapper = document.querySelector("#register-fields");
 const boardWrapper = document.querySelector(".account__board-wrapper");
 const userDashboard = document.querySelector(".account__board-user");
 const welcomeSplash = document.querySelector('.account__splash-img');
 const accountSection =document.querySelector(".account");
 const fields = ['name','phone'];
-
+const registerFieldsIds = ['email','password','password_confirmation'];
 /*STRINGS*/
 const SIGN_IN = 'Sign In';
-registerFields.remove();
-boardWrapper.remove();
+const REGISTER = 'Register';
+
+const addRegisterFieldsToValidator = () =>{
+    // First step, combine the actual validator fields with the registerFields
+    const fieldsCombined = [...validator.fields,...registerFieldsIds];
+    // We have to filter the array to delete the repeated elements 
+    const updatedFormFields = fieldsCombined.filter((field,i)=> fieldsCombined.indexOf(field) === i);
```

Comment:
if the only thing we are doing here is removing duplicates then you can use `new Set` as you did before only following the advice seen here in the 2nd answer https://stackoverflow.com/questions/9229645/remove-duplicate-values-from-js-array/9229821